frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -44,6 +48,10 @@ const initialState: DashState = {
   projectData: {},
 };
 
+const initialRefreshState: RefreshRateState = {
+  refreshRate: 30000,
```

Comment:
can a comment be added that this is 30 seconds since the unit might not be obvious